backend/inspirehep/utils.py

```diff
@@ -24,7 +24,11 @@
 
 def send_zulip_notification(message: str):
     try:
-        zulip_client = zulip.Client()
+        zulip_client = zulip.Client(
+            api_key=os.environ.get("ZULIP_NOTIFICATION_API_KEY"),
+            email=os.environ.get("ZULIP_NOTIFICATION_EMAIL"),
+            site=os.environ.get("ZULIP_NOTIFICATION_SITE"),
+        )
```

Comment:
As we want this to run only in PROD, should we exclude adding the config on qa -> https://github.com/cern-sis/kubernetes-inspire/pull/246?

Reply:
It's not a bad idea, in that case I guess we can send the messages to inspire-qa-tasks. WDYT @drjova ? Would QA make sense in your opinion ?